core/src/main/java/com/linecorp/armeria/client/Http1ResponseDecoder.java

```diff
@@ -171,8 +173,14 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
         }
         keepAliveHandler.onReadOrWrite();
 
+        if (state == State.DISCARD_DATA_OR_TRAILERS && msg instanceof HttpResponse) {
```

Comment:
Question) Just imagining scenarios, but if the 417 request included a trailer in the response could the response for the repeated request be corrupted?

Reply:
The `LastHttpContent` that has the trailing headers will be ignored.